modules/sql/src/main/resources/db/migration/V066__Add_Gsaoi.sql

```diff
@@ -0,0 +1,151 @@
+--
+-- Name: e_gsaoi_read_mode; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE e_gsaoi_read_mode (
+    id                        identifier            PRIMARY KEY,
+    short_name                character varying(8)  NOT NULL,
+    long_name                 character varying(40) NOT NULL,
+    ndr                       smallint              NOT NULL CHECK (ndr >= 0),
+    read_noise                smallint              NOT NULL CHECK (read_noise >= 0),
+    minimum_exposure_time     numeric(3,1)          NOT NULL CHECK (minimum_exposure_time >= 0),
+    overhead                  smallint              NOT NULL CHECK (overhead >= 0)
+);
+
+ALTER TABLE e_gsaoi_read_mode OWNER TO postgres;
+
+--
+-- Data for Name: e_gsaoi_read_mode; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY e_gsaoi_read_mode(id, short_name, long_name, ndr, read_noise, minimum_exposure_time, overhead) FROM stdin;
+Bright	Bright	Bright Objects	2	28	5.3	10
```

Comment:
Yeah it looks like the 300 ms are being rounded down in the conversion to `FiniteDuration`.